build/demo-launcher/src/launcher/static/index.html

```diff
@@ -0,0 +1,403 @@
+                    </div>
+
+                    <!-- Private Key -->
+                    <div class="col-span-6 sm:col-span-4">
+                        <label for="username" class="class="block text-sm font-medium text-gray-700"">Private key
+                            <span class="italic text-red-400">*</span>
+                        </label>
+                        
+                        <textarea id="private_key"  rows="4" cols="50" placeholder="Enter SSH private key" class="border-gray-300 focus:border-blue-300 focus:ring-blue-300 mt-1 block w-full rounded-md border py-2 px-3 shadow-sm sm:text-sm focus:outline-none" required></textarea>
+                    </div>
+
+                    <!-- Public Certificate -->
+                    <div class="col-span-6 sm:col-span-4">
+                        <label for="username" class="class="block text-sm font-medium text-gray-700"">Public certificate
+                            <span class="italic text-gray-400">Optional</span>
+                        </label>
+                        
+                        <textarea id="public_cert"  rows="4" cols="50" placeholder="Enter SSH public certificate" class="border-gray-300 focus:border-blue-300 focus:ring-blue-300 mt-1 block w-full rounded-md border py-2 px-3 shadow-sm sm:text-sm focus:outline-none"></textarea>
+                    </div>
+
+                </div>
+                <!-- Footer -->
+                <div class="items-center justify-end bg-gray-50 mt-6 px-4 py-3 text-right sm:px-6">
+                    <button id="credentials_button" type="button" onclick="submitCredentials()" class="h-9 inline-flex justify-center rounded-md border border-transparent bg-emerald-500 py-2 px-4 text-sm font-medium text-white shadow-sm hover:bg-emerald-600 focus:outline-none focus:ring-2 focus:ring-emerald-600 focus:ring-offset-2">
+                        Next
+                    </button>
+                </div>
+            </div>
+        </div>
+
+
+
+     
+
+        <!-- Card Connection -->
+        <div id="step2" class="step bg-white shadow rounded-lg divide-y divide-gray-200 mb-4">
+            <!-- Title -->
+            <div class="px-4 sm:px-6 pt-[13px] pb-[13px]">
+                <div class="-ml-4 -mt-4 flex justify-between items-center flex-wrap sm:flex-nowrap">
+                    <div class="ml-4 mt-4">
+                        <h3 class="text-lg leading-6 font-medium text-gray-900">Step 2: Set-up your SSH Connection</h3>
+                    </div>
+                </div>
+            </div>
+
+            <!-- Form Body -->
+            <div class="px-4 py-5 sm:p-6">
+
+                <p class="mb-6">Provide your cluster’s login node hostname and SSH port.<br/>If a SSH tunneling is required please provide the proxy hostname and port.</p>
```

Comment:
As discussed, allow also a "public name" field to name the cluster besides the FDQN

Reply:
sure, done!